optionlab/support.py

```diff
@@ -151,6 +154,7 @@ def create_price_seq(min_price: float, max_price: float) -> np.ndarray:
         raise ValueError("Maximum price cannot be less than minimum price!")
 
 
+@lru_cache
```

Comment:
create_sequential_prices() should also be cached.

Reply:
couldn't find this anywhere in the codebase

Reply:
ah `create_price_seq`. yep found it.